app/views/donations/_table.html.erb

```diff
@@ -0,0 +1,37 @@
+<table class="table table-striped table-responsive data-table">
+  <thead>
+    <tr>
+      <th class="col-xs-1">ID</th>
+      <th class="col-xs-2">Donor</th>
+      <th class="col-xs-2">Items</th>
+      <th class="col-xs-2">Value</th>
+      <th class="col-xs-2">Date Received</th>
+
+      <% if local_assigns[:for_deleted] %>
+        <th class="col-xs-2">Actions</th>
+      <% else %>
+        <th>Notes</th>
+      <% end %>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @donations.each do |donation| %>
+      <tr <%= "data-href=\"#{sanitize donation_path(donation)}\"".html_safe unless local_assigns[:for_deleted] %>>
+        <td><%= donation.id %></td>
```

Comment:
Does Grateful Garments care about this number, thinking we might want to remove?

Reply:
will there be a number from integration that is more applicable?

Reply:
Ignore this

Reply:
Resolved in meeting